app/controllers/candidate_interface/application_review_controller.rb

```diff
@@ -1,7 +1,5 @@
 module CandidateInterface
```

Comment:
Is this class still used?

Reply:
no, we do not need it, that was part of the refactor, we do not need anymore, thanks.